src/includes/getting-started-install/native.mdx

```diff
@@ -6,19 +6,20 @@ For example, `CMake` can be used like this (on macOS):
 
 ```shell
 # configure the cmake build into the `build` directory, with crashpad (on macOS)
-cmake -B build -D SENTRY_BACKEND=crashpad --config RelWithDebInfo 
+cmake -B build -D SENTRY_BACKEND=crashpad --config RelWithDebInfo -S .
 # build the project
 cmake --build build --config RelWithDebInfo --parallel
 # install the resulting artifacts into a specific prefix
 cmake --install build --prefix install
 # which will result in the following (on macOS):
-exa --tree install
+exa --tree install --level 2
 install
 ├── bin
 │  └── crashpad_handler
 ├── include
 │  └── sentry.h
 └── lib
+   ├── cmake
```

Comment:
the link on line 31 is missing the preceding `/` - should be `/configuration/backends/crashpad/`

Reply:
Not part of this patch, but I can add a drive-by fix for it :)